test/integration/edition_edit_test.rb

```diff
@@ -1995,6 +2042,29 @@ class EditionEditTest < IntegrationTest
           assert page.has_no_link?("Cancel scheduling")
         end
       end
+
+      context "scheduled_for_publishing place edition" do
```

Comment:
Given we're already in a context of "scheduled_for_publishing edition", we don't really need to duplicate that here.
```suggestion
      context "place edition" do
```